suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -440,7 +446,12 @@ const discoverNetworkBatchThunk = createThunk(
         const chunkBundle: Array<DiscoveryItem> = [];
 
         A.makeWithIndex(batchSize, batchIndex => {
-            const index = lastDiscoveredAccountIndex + batchIndex;
+            const isEvmLedgerDerivationPath =
+                network.networkType === 'ethereum' && network.accountType === 'ledger';
+            // first Ledger derivation path for EVM networks is the same as trezor, so we need to skip it (consistent with desktop)
+            const index =
+                lastDiscoveredAccountIndex + batchIndex + (isEvmLedgerDerivationPath ? 1 : 0);
+
             const accountPath = network.bip43Path.replace('i', index.toString());
 
             const isAccountAlreadyDiscovered = selectIsAccountAlreadyDiscovered(getState(), {
```

Comment:
it speeds up discovery of this network in case it was interrupted and not finished, but device is in view only. In such case, already discovered accounts are not fetched again.